src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -1758,124 +1753,9 @@ public void shouldReturnInstanceWhenFilterByTags() throws Exception {
 
   }
 
-  @Test
-  public void canCreateCollectionOfInstances()
-    throws InterruptedException, ExecutionException, TimeoutException {
-
-    JsonArray instancesArray = new JsonArray();
-    int numberOfInstances = 1000;
-
-    for (int i = 0; i < numberOfInstances; i++) {
-      instancesArray.add(smallAngryPlanet(UUID.randomUUID()));
-    }
-
-    JsonObject instanceCollection = new JsonObject()
-      .put(INSTANCES_KEY, instancesArray)
-      .put(TOTAL_RECORDS_KEY, numberOfInstances);
-
-    CompletableFuture<Response> createCompleted = new CompletableFuture<>();
-
-    getClient().post(instancesStorageBatchInstancesUrl(StringUtils.EMPTY), instanceCollection, TENANT_ID,
-      json(createCompleted));
-
-    Response response = createCompleted.get(30, SECONDS);
-
-    assertThat(response.getStatusCode(), is(HttpURLConnection.HTTP_CREATED));
-
-    JsonObject instancesResponse = response.getJson();
-
-    assertThat(instancesResponse.getInteger(TOTAL_RECORDS_KEY), is(numberOfInstances));
-
-    JsonArray instances = instancesResponse.getJsonArray(INSTANCES_KEY);
-    assertThat(instances.size(), is(numberOfInstances));
-    assertThat(instances.getJsonObject(1).getJsonObject(METADATA_KEY), notNullValue());
-    assertThat(instances.getJsonObject(1).getString(STATUS_UPDATED_DATE_PROPERTY), hasIsoFormat());
-
-    assertNotSuppressedFromDiscovery(instances);
-
-    instanceMessageChecks.createdMessagesPublished(toList(instances));
-  }
-
   // Interesting Times has two ISBNs: 0552167541, 978-0-552-16754-3
   // and an invalid ISBNs: 1-2-3-4-5
 
-  @Test
```

Comment:
I think the comment was related to canCreateInstancesEvenIfSomeFailed test, so could be removed.